src/lwc/rd2EditPaymentInformationModal/rd2EditPaymentInformationModal.html

```diff
@@ -0,0 +1,77 @@
+<template>
+    <div>
+        <section 
+            role="dialog"
+            tabindex="-1"
+            aria-labelledby="modal-heading-01"
+            aria-modal="true"
+            aria-describedby="modal-content-id-1"
+            class="slds-modal slds-fade-in-open slds-modal_small"
+            onkeyup={handleKeyUp}>
+            <div class="slds-modal__container">
+                <header class="slds-modal__header slds-clearfix">
+                    <lightning-button-icon
+                        class="slds-modal__close"
+                        title="Close"
+                        icon-name="utility:close"
+                        icon-class="slds-button_icon-inverse"
+                        onclick={handleClose}
+                        onkeydown={handleClosedButtonTrapFocus}
+                        data-id="closeButton"
+                        disabled={isSaveButtonDisabled}>
+                    </lightning-button-icon>
+                    <h2
+                        class="slds-text-heading_medium slds-hyphenate header-string slds-p-top_large slds-p-horizontal_medium" id="modal-heading-01">
+                        {labels.PaymentInformationTitle}
+                    </h2>
+                </header>
+                <div class="slds-modal__content slds-p-top_none slds-p-horizontal_medium slds-p-bottom_medium" id="modal-content-id-1">
+
```

Comment:
If there is a blank line before the template tag, there should probably be one after as well.  Either with or without both works.